### DISCUSSION TITLE
cannot partially overlap loop

### DISCUSSION THREAD

**Discussion by IngridJSJ** [Spine-project/Spine-Toolbox#1968] [Support]
Dear Spine team, 
I would like to make an initialization loop of the modules 16, 17 and 18 before the main loop. 
So that these are run some times as modules 6, 7, 8 
But when adding the loop I get the messsage: 
from 8 to 6 cannot partially overlap from 18 to 16. 
I dont understand why this error comes in this loop and not in other loops
Thank you for any hints

**Comment by soininen** [comment]
Hi Ingrid! I'm not sure I understand how your loops are organized. Would it be possible you shared a screenshot of the workflow?

**Comment by IngridJSJ** [comment]
![Capture](https://user-images.githubusercontent.com/47012366/218781572-f5ec0f80-471d-4a5a-838d-200cd23b9569.jpg)
Hi Annti, 
so loop 1 is the same as loop 2 (magenta) but should only be executed once, before the main loop (orange)
Without loop 1 the workflow works fine, but when I add it then there is this blue exclamation symbol

**Reply by soininen** [reply]
Thank you for the screenshot! I can now reproduce the issue here. It does look like a bug but I'll need to investigate further.

**Comment by soininen** [comment]
This indeed turned out to be a bug which has been fixed now. Please, update Toolbox requirements.
